src/App.vue

```diff
@@ -384,6 +386,10 @@ export default class App extends Mixins(
       this.fileAndPayInvalidNameRequestDialog
     )
   }
+  /** The Fee Summary filing text for firm businesses. */
+  get filingLabelText (): string {
+    return this.isTypeFirm && 'Dissolution'
```

Comment:
What's the default filing fee label here?

Why not create a new fee code with the desired label?

Reply:
Only a label change for SP/GP, fee code should stay the same and be treated as a voluntary dissolution as per Jyoti's instruction.